R/MAE_to_PSet.R

```diff
@@ -0,0 +1,163 @@
+#' Convert MultiAssayExperiment to TreatmentResponseExperiment
+#'
+#' This function converts a MultiAssayExperiment generated by gDR into a TreatmentResponseExperiment for use in the
+#' PharmacoGx package. The resulting PharmacoSet can be used for pharmacogenomic analysis of drug response.
+#'
+#' @param mae A MultiAssayExperiment object generated by gDR.
+#' @param pset_name A character string specifying the name of the resulting PharmacoSet object.
+#' 
+#' @return A PharmacoSet object.
+#' 
+#' @importFrom checkmate assert_class assert_string
```

Comment:
instead of using `importFrom` please use `package::function` namespacing in the code

Reply:
Thanks Bartek, I have made those changes.